api/answer_check.py

```
@@ -1,10 +1,21 @@
-    _t = cut(answer)
-    if _t is not None and len(_t) == 1:
-        return True
-    else:
+    if answer is None:
         return False
 
+    text = str(answer).strip()
+    if not text:
+        return False
+
+    # 单选答案文本中常见逗号（中英文）是句内标点，不应据此判定为多选。
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)